apps/demo/app/custom-ui/[...puckPath]/client.tsx

```diff
@@ -276,6 +276,35 @@ const CustomPuck = ({ dataKey }: { dataKey: string }) => {
   );
 };
 
+const CustomActionBar = ({ children, label }) => {
+  const { appState } = usePuck();
```

Comment:
You can use `selectedItem` from usePuck instead of reaching into the state yourself

```suggestion
  const { appState, selectedItem } = usePuck();
```